api/apps/api/src/modules/scenarios/scenarios.controller.ts

```diff
@@ -399,7 +401,7 @@ export class ScenariosController {
   @ApiConsumesShapefile({ withGeoJsonResponse: false })
   @GeometryFileInterceptor(GeometryKind.ComplexWithProperties)
   @ApiTags(asyncJobTag)
-  @Get(`:scenarioId/cost-surface/shapefile`)
+  @Post(`:scenarioId/cost-surface/shapefile`)
```

Comment:
Hey, I am not sure if this is correct. @andresgnlez mentioned that this should be a GET endpoint, I rolled it back in 7d12f1dbf03a6a7e89f64eca19ce9eed246af1dc

We mentioned that in future implementations (`ProjectCostSurfaceController`) since this has to receive a name, we could probably set it as POST, however since this is a deprecated endpoint, probably should be GET

Reply:
there was a bit of confusion here, because @andresgnlez reported that the endpoint of `GET :id/cost-surface` was returning 404 - he meant the one without shapefile in the end and the one that return cost surface range (hat was removed and not rolled back).